packages/terra-abstract-modal/src/AbstractModal.jsx

```diff
@@ -65,6 +73,10 @@ const propTypes = {
    * Callback function to set the reference of the element that will receive focus when the Slide content is visible.
    */
   setModalFocusElementRef: PropTypes.func,
+  /**
+   * If set to true, the AbstractModal is rendered inside a NotificationDialog.
+   */
+  isCalledFromNotificationDialog: PropTypes.bool,
```

Comment:
let's make this prop private, As it's used internally on notification-dialog

Reply:
Done [71cf564](https://github.com/cerner/terra-framework/pull/1894/commits/71cf564a7392da594d93aead47c597ef34f9a613)